### PR TITLE
Add phone_number, customer_name and risk

### PR DESCRIPTION
**Pull request by justingrobles**
## Description
Add phone_number, customer_name and risk

## Reference
- [Trello Card](https://trello.com/c/qrs6MZRw)
- [ActiveMerchant code for phone number](https://github.com/activemerchant/active_merchant/pull/4800)

### CODE REVIEW

File: test/unit/gateways/checkout_v2_test.rb [hunk 1]
```diff
@@ -455,6 +456,52 @@ def test_successful_purchase_with_stored_credentials_merchant_initiated_transact
...
+      refute_includes data, 'name'
+    end.respond_with(successful_purchase_response)
+  end
+
+  def test_successful_purchase_with_risk_data
+    stub_comms(@gateway, :ssl_request) do
+      options = {
+        risk: {
+          enabled: true,
+          device_session_id: dsid_ipsmclhxwq72phhr32iwfvrflm
```
Comment (bwerdschinski) [inline test/unit/gateways/checkout_v2_test.rb:494]: Small typo here, you'll need to wrap this in quotes for the tests to pass 🙂

Comment (justingrobles) [reply test/unit/gateways/checkout_v2_test.rb]: Done. Thanks @bwerdschinski.

**Review by bwerdschinski — APPROVED** [review]
👍

**Comment by damienjbyrne** [comment]
@justingrobles specs are failing here, looks like they run with github actions.  Some clicking around indicates that maybe it's to do with this: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

But could be to do with ruby versions too.

**Comment by damienjbyrne** [comment]
Maybe we need to update our `.github/workflows/ruby-ci.yml` to match the parent repo?

**Comment by justingrobles** [comment]
image
Probably Ruby.

**Comment by justingrobles** [comment]
Would it be much better to
- Push Marketplace changes (https://trello.com/c/tbuW847B/3570-5-contribute-back-to-activemerchant) to ActiveMerchant Public Repo
- Update our Fork
- Then test this branch

**Comment by damienjbyrne** [comment]
> Maybe we need to update our .github/workflows/ruby-ci.yml to match the parent repo?

Seems the parent repo doesn't care about passing tests...
image

**Review by bwerdschinski — APPROVED** [review]
👍

**Comment by justingrobles** [comment]
Closing this as changed the approach on the Comment under [Trello Card](https://trello.com/c/qrs6MZRw/4378-activemerchantold-add-phonenumber-and-risk)

**Pull request closed without merging**